lib/engine/src/artifact.rs

```diff
@@ -179,7 +179,7 @@ pub trait Artifact: Send + Sync + Upcastable + MemoryUsage {
 }
 
 // Implementation of `Upcastable` taken from https://users.rust-lang.org/t/why-does-downcasting-not-work-for-subtraits/33286/7 .
-/// Trait needed to get downcasting from `WasiFile` to work.
+/// Trait needed to get downcasting from `VirtualFile` to work.
```

Comment:
Is this correct? Engines should not care about `WasiFile` or `VirtualFile`

Reply:
Looks like the doc comment was out of date, we needed the same pattern for engines too, I did a bulk find and replace for `WasiFile` -> `VirtualFile` and didn't realize this file had been edited. We should fix up this doc comment to be about engines